ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -0,0 +1,22 @@
+---
+# This task will install the standard actions and packages available in openwhisk-catalog repos.
+
+- set_fact:
+    catalog_location={{ openwhisk_home }}/../{{ item.key }}
```

Comment:
I think the user should be able to put openwhisk-catalog where they want.  We could use an environment variable here that points to the user's openwhisk-catalog.